pkg/kubelet/kubelet_network.go

```diff
@@ -19,10 +19,10 @@ package kubelet
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
vscode linter usually adds this v1, would suggest to remove it and keep as the old entry